kotlin/add-features-with-contingent-values/src/main/res/layout/activity_main.xml

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="utf-8"?>
+<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    tools:context="com.esri.arcgisruntime.sample.addfeatureswithcontingentvalues.MainActivity">
+
+    <com.esri.arcgisruntime.mapping.view.MapView
```

Comment:
Generally, I use ConstraintLayout as the root element due to its [low cost](https://developer.android.com/topic/performance/rendering/optimizing-view-hierarchies#cheaper) while drawing the root view. 

Although using just the MapView would not affect load times much, should we keep this practice as standard from now on? And if we will, would it be worth looking into previous sample implementations since there would be a multiple samples with such an approach.